src/main/java/seedu/address/logic/commands/FilterTransCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_CLIENTS;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Filters the all the transaction by all the clients.
```

Comment:
Perhaps this sentence should be 'Filters all the transaction by all the clients.` instead?